docs/integrations/datadog.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Datadog
+---
+
+### Overview
+
+There are two aspects to the Datadog integration, forwarding event counts and change config changes. The goal of this integration is to allow monitoring of Statsig usage for billing purposes and alerts for config changes. 
+We also offer a Datadog Trigger integration which you can read more about [here](https://docs.statsig.com/integrations/triggers/Datadog)
+
+### Connecting to Datadog
+
+<ol>
+  <li>
```

Comment:
these wont work in .md, only in .mdx

But I think you probably copied from console and dont need this? You can use markdown syntax for lists, for example

Reply:
It does actually render as expected, I was just to lazy to convert but i will